### DISCUSSION TITLE
ideavim 2.8.0 "source ~/.vim-mapping" command does not take effect

### DISCUSSION THREAD

**Discussion by xkia** [JetBrains/ideavim#810] [Q&A]
The local configuration file is split into multiple configuration files and summarized through the "source" of the main file. The current command does not take effect.

<img width="479" alt="image" src="https://github.com/JetBrains/ideavim/assets/5953602/32245eb9-71b6-4154-be03-492ea37ac7a4">

**Comment by citizenmatt** [comment]
Please try 2.8.1, it should fix this issue
